packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -454,7 +454,7 @@ export class RunningSummarizer implements IDisposable {
 			if (this.summarizingLock === undefined) {
 				this.trySummarizeOnce(
 					// summarizeProps
-					{ reason: "lastSummary" },
+					{ summarizeReason: "lastSummary" },
```

Comment:
Does this need to be under `details.summarizeReason`? I thought we were already past the limit of columns in Kusto so any new properties just won't show up (@markfields ? ).

Reply:
If so we can get it from `Payload`. It's possible it's actually there already though.